src/components/TermsOfService.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <v-container>
+    <v-layout justify-center>
+      <v-flex md6>
+        <h2>DD-DeCaF Terms of Service</h2>
```

Comment:
We should probably use the [vuetify typography classes](https://vuetifyjs.com/en/framework/typography) here - but if not, at least use `h1` like on the other views for a consistent header size.